kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -185,7 +185,8 @@ private[kyuubi] class EngineRef(
 
     MetricsSystem.tracing(_.incCount(ENGINE_TOTAL))
     try {
-      info(s"Launching engine:\n$builder")
+      val redactedCmd = Utils.redact(conf.get(SERVER_SECRET_REDACTION_PATTERN), builder.toString)
```

Comment:
shall we redact it before combine the submission cmd ？so we can redact the value inside `key=value`.

Reply:
make sense, I try another way.